packages/cadl-ranch-specs/http/authentication/apiKey/main.cadl

```diff
@@ -0,0 +1,30 @@
+import "@cadl-lang/rest";
+import "@azure-tools/cadl-azure-core";
+import "@azure-tools/cadl-ranch-expect";
+
+using Cadl.Http;
+using Azure.Core.Foundations;
+
+@serviceTitle("ApiKeyAuthentication")
+@serviceVersion("1.0.0")
+@server("http://localhost:3000", "TestServer endpoint")
+@route("/authentication/api-key")
+@doc("Illustrates clients generated with ApiKey authentication.")
+@useAuth(ApiKeyAuth<ApiKeyLocation.header, "x-ms-api-key">)
+namespace ApiKeyAuthentication;
+
+interface ApiKey {
```

Comment:
Do we need this interface? I think just having the operation at the root would make the scenario name cleaner

Reply:
Sure, I will remove them (was following the "extensible-enums" file which having a String interface).

Reply:
Yeah I did that there because I think we could have other extensible enum types in the future. But here I think this is just redundant.

Reply:
I think in general all specs  don't need to have the exact same hierachy. Some are going to need more or less nesting